utils/generate_types/index.js

```diff
@@ -184,7 +184,8 @@ function createEventDescriptions(classDesc) {
   if (!hasUniqueEvents(classDesc))
     return [];
   const descriptions = [];
-  for (const [eventName, value] of classDesc.events) {
+  for (let [eventName, value] of classDesc.events) {
```

Comment:
Is there a corresponding change in playwirght.dev?

Reply:
There is now: https://github.com/microsoft/playwright.dev/pull/104 :-)